feat(oneaccount): proposal to clarify (#5760)
scaleway/docs-content: pages/dedibox-account/how-to/disable-two-factor-authentication.mdx

You can disable the multifactor authentication of your Dedibox account if you want to register a new authentication device.

<Message type="important">

>   This page applies only for users with no Dedibox and Scaleway unified account. If your Dedibox and Scaleway accounts have been unfied, you can [read more about multifactor authentication at this page](account/how-to/use-2fa/)

Reviewer: ```suggestion
  This page applies only to users who do not have a unified Dedibox and Scaleway account. If your Dedibox and Scaleway accounts have already been unified, you can [learn more about multifactor authentication on this page](account/how-to/use-2fa/).
```
